clang/test/CodeGen/arm-v8.1m-check-mve.c

```diff
@@ -0,0 +1,35 @@
+// RUN: %clang --target=arm-none-eabi -mcpu=cortex-m85 -mfloat-abi=hard -O2 -save-temps=obj -S -o - %s | FileCheck %s
```

Comment:
Should there be an m55 test for this as well, as you've got 55 and 85 for save-temps.

Reply:
Done